packages/admin-panel/src/pages/resources/CountriesPage.jsx

```diff
@@ -29,9 +29,9 @@ const EXPANSION_CONFIG = [
 ];
 
 const CREATE_CONFIG = {
-  title: 'New Country',
+  title: 'New country',
   actionConfig: {
-    title: 'Create New Country',
+    title: 'New country',
```

Comment:
The majority of these `createConfig`s have a `title` _outside_ of `actionConfig`, but I can’t actually see where they get used. They don’t appear in the final “compiled” DOM anywhere

Any idea what this does? If nothing, then my inclination would be to remove all of them, keeping only `actionConfig.title`, which is the actual string used as the title in modals

Reply:
My suspicion is that it doesn't do anything when outside of `actionConfig`, and this might be a hangover from an older way of doing things. That's probably why the correct titles weren't showing

Reply:
Strange because `createConfig` is the only one that behaves this way 🤔

It’s only now occurred to me to check Git blame/history to see which behaviour is newer